policyengine_us/variables/gov/states/az/tax/income/credits/property_tax_credit/az_property_tax_credit.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class az_property_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona Property Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = "az_property_tax_credit_eligible"
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(period).gov.states.az.tax.income.property_tax_credits
+        income = tax_unit("az_property_tax_credit_income", period)
+
+        cohabitating = tax_unit("cohabitating_spouses", period)
+
+        return select(
```

Comment:
https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2022_140PTC-2D.pdf#page=7

The credit should be capped at rent and the real_estate_taxes, summed up